net/api/impl/posix.h

```diff
@@ -87,7 +89,7 @@ class HTTPClientPOSIX final {
       // connection.SendEOF();
       message_.reset(new HTTPRedirectableReceivedMessage(connection));
       // TODO(dkorolev): Rename `URL()`, it's only called so now because of HTTP request/response format.
-      const int response_code_as_int = atoi(message_->URL().c_str());
+      const int response_code_as_int = atoi(message_->Path().c_str());
```

Comment:
It just extracts the second part of the first non-empty HTTP line.

Where the request contains ```GET /path HTTP/1.1```, making ```"/path"``` a path, the response contains ```HTTP/1.1 200 OK```. Thus, the ```200``` part in the response is located exactly same way as the ```"/path"``` part is located in the request, thus variable/method name reuse.

I'll clean it up one day.